src/services/Credential.js

```diff
@@ -506,20 +483,22 @@ class Credential {
 		return this.hasKey(key) ? (this[key] || this[key.toLowerCase()]) : null;
 	}
 
+	get hasPrivateKey() {
+		return this.hasKey("PRIVATE_KEY");
+	}
+
+	get getPrivateKey() {
```

Comment:
`getPrivateKey` should be `privateKey`. It's getter so should be named like a property.

Reply:
Property is accessible directly, so no need for the getter.